packages/cdc-react/src/components/ProgressTracker/ProgressTracker.tsx

```diff
@@ -0,0 +1,62 @@
+import "./ProgressTracker.scss";
+
+export interface ProgressTrackerProps {
+  isComplete?: boolean;
+  currentAmount?: number;
+  isIndeterminate?: boolean;
+  label?: string;
+  totalAmount?: number;
+}
+
+export const ProgressTracker = ({
+  isComplete = false,
+  currentAmount = 0,
+  isIndeterminate = false,
+  label = "",
+  totalAmount = 0,
+}: ProgressTrackerProps) => {
+  const getBarClass = () => {
+    const classList = ["progress-bar-fill"];
+    if (isComplete) classList.push("complete");
+    if (isIndeterminate) classList.push("indeterminate");
+
+    return classList.join(" ");
+  };
+
+  const getBarPercentage = (): number => {
+    if (isIndeterminate || isComplete) return 100;
+
+    currentAmount = Math.max(0, Math.min(currentAmount, totalAmount));
+    totalAmount = Math.max(0, totalAmount);
+
+    const percentage =
+      totalAmount > 0 ? (currentAmount / totalAmount) * 100 : 0;
+
+    return Math.floor(percentage);
+  };
+
+  const getBarPercentageInfo = (): string => {
+    if (isComplete) return "Complete";
+    if (isIndeterminate) return "In progress...";
+    return `${getBarPercentage()}% complete`;
```

Comment:
A wrapper method to slice the string max to show two decimals would be useful here I think.